lib/market_open_days/nyse_market_days.csv

```diff
@@ -1737,11 +1737,11 @@
 2020-11-16
 2020-11-17
 2020-11-18
+2020-11-19
 2020-11-20
 2020-11-23
 2020-11-24
 2020-11-25
-2020-11-26
```

Comment:
Bruh these dates are correct!
11/28/2019 - SPY closed at 310.27, was closed on the 28th
11/19/2020 - SPY closed at 357.78, was closed on the 26th

![image](https://github.com/wakproductions/stock_market_days/assets/1912237/4e6b8a8e-fcd6-4b0b-95bb-3523dd6ffebd)


![image](https://github.com/wakproductions/stock_market_days/assets/1912237/06c362ad-5852-42d9-b217-94a0011dd6d2)

Reply:
oh darn that's totally right! what was I thinking?! thanks so much for your contribution!!! sorry for the misunderstanding.